source/features/previous-version.tsx

```diff
@@ -0,0 +1,117 @@
+import * as pageDetect from 'github-url-detection';
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+
+import features from '../feature-manager';
+import * as api from '../github-helpers/api';
+import observe from '../helpers/selector-observer';
+
+import GitHubURL from '../github-helpers/github-url';
+
+const getHistoryOids = cache.function('previous-version', async (githubUrl: GitHubURL): Promise<string[] | false> => {
+	const {resource: {history}} = await api.v4(`
```

Comment:
You can do `repository()` and then use expression

No need to specify the URL in full

Reply:
I'm not sure that helps, also due to future GraphQL improvements I mentioned before

Reply:
Yes, But in this case I need to use `resource` query not `repository` query.
So I'll just preserve it.